projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -3555,7 +3555,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         });
 
         this.verticalScrollContainer.contentSizeChange.pipe(filter(() => !this._init), destructor).subscribe(() => {
```

Comment:
Shouldn't there be a throttleTime here as well, like in the PRs to 13.2.x and 14.0.x?